thinkbox-deadline/azuredeploy.json

```diff
@@ -56,10 +65,29 @@
             }
         },
 
+        "repositorySize":
+        {
+            "type": "string",
+            "defaultValue": "Standard_A1",
+            "metadata":
+            {
+                "description": "Size of the repository."
+            }
+        },
+
+        "slaveSize":

```

Comment:
For clarity sake, it'd be better to call this slaveVMSize so the user knows what to input while deploying without looking at the template